### PR TITLE
track and report container state correctly

### PR DESCRIPTION
**Pull request by bergwolf**
containerd-shim will query container status before deleting it.

### CODE REVIEW

File: cli/container.go [hunk 1]
```diff
@@ -110,6 +118,7 @@ func createContainer(options runvOptions, vm *hypervisor.Vm, container, bundle,
 			Pid:     shim.Pid,
 			Bundle:  bundle,
 		},
+		stat.StartTime, 0,
```
Comment (laijs) [inline cli/container.go]: please use ShimCreateTime: stat.StartTime

Comment (bergwolf) [inline cli/container.go]: golang does not allow mixing field:value and value initializers. e.g.,
`cli/container.go:114: mixture of field:value and value initializers`

Comment (bergwolf) [inline cli/container.go]: You were right. I've fixed it.

**Comment by laijs** [comment]
LGTM

**Pull request merged**